src/classmentors/components/events/events.js

```diff
@@ -2112,7 +2111,7 @@ function ClmEventTableCtrl($scope, $q, $log, $mdDialog, $document,
         var parent = this;
 
         function loadEditor() {
-            var editor = ace.edit($document.querySelector('#editor'));
+            var editor = ace.edit(document.querySelector('#editor'));
```

Comment:
Will attempt to do so. Out of curiosity, is there a reason why you recommend this?
"_you should avoid manipulate the DOM from a controller_"